Test.md

```diff
@@ -0,0 +1 @@
+![icon](https://octodex.github.com/images/justicetocat.jpg)
```

Comment:
Great use of the including an image to a file. Perhaps try to include a link to a website like, github.com , or creating a table using markdown 👍